airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/products_report.py

```diff
@@ -263,6 +280,10 @@ def _get_init_report_body(self, report_date: str, record_type: str, profile):
             if profile.accountInfo.type == "vendor":
                 metrics_list = copy(metrics_list)
                 metrics_list.remove("sku")
+        elif record_type == RecordType.PRODUCTADS and profile.accountInfo.type != "seller":
+            # Remove SKU from metrics since it is only available for seller accounts in Product Ad report
+            metrics_list = copy(metrics_list)
```

Comment:
```suggestion
            metrics_list = [m for m in metrics_list if m != "sku"]
```
List comprehension would be more pythonic since it constructs a new list in one pass and doesn't mutate the copy of the list

Reply:
@monai Thanks for the comment. I have updated the code.